smartcontractkit/external-adapters-js
# Galaxis NBA

## Closes 32011

## Description

This PR contains the Galaxis NBA EA implementation

## Changes

- Implement Galaxis EA NBA endpoint

## Steps to Test

1. Run EA by following README instructions
2. Send sample request e.g
```
{
  "id": "1",
  "data": {}
}
```
3. Verify successful response and that response is valid
4. Integration test for a successful API response and that the encoded value is decoded to the expected variables

## Quality Assurance


Future Work:

 - Expand tests: https://app.shortcut.com/chainlinklabs/story/33777/unit-tests-for-galaxis-ea
